### PR TITLE
Chapter 5, task 5.9

### PR DESCRIPTION
**Pull request by alexVOLTS**
Task #5.9 is ready to check

### CODE REVIEW

File: Chapter_5/7.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <stdio.h>
+#include <locale.h>
+#include <float.h>
+
+double x;
+
+int calculate ()
```
Comment (dimmykar) [inline Chapter_5/7.c:7]: bleat'

File: Chapter_5/7.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{
+    printf("\nInput number with double type: ");
+    scanf("%lf", &x);
+    x = x * x * x;
+    return x;
+}
+
+int main (void)
```
Comment (dimmykar) [inline Chapter_5/7.c:15]: ditto

File: Chapter_5/8.c [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+int first ()
+{
+    printf("\nInput first operand (<=0 to exit from program): ");
+    scanf("%d", &y);
+    return y;
+}
+
+int cycle (void)
```
Comment (dimmykar) [inline Chapter_5/8.c:22]: remove space, please

File: Chapter_5/9.c [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+double x;
+double y;
+double temp;
+int scanfresult;
+const double kelvin = 273.16;
+const double f1 = 5.0 / 9.0;
+const double f2 = 32.0;
+
+
+int Temperatures()
```
Comment (dimmykar) [inline Chapter_5/9.c:14]: Missed `void`

File: Chapter_5/9.c [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+{
+    double x = f1 * (temp - f2);
+    double y = x + kelvin;
+    printf("\nTemperature Celsium = %.2lf", x);
+    printf("\nTemperature Kelvin = %.2lf", y);
+}
+
+int Input ()
```
Comment (dimmykar) [inline Chapter_5/9.c:22]: ditto

File: Chapter_5/9.c [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+{
+    printf("\nInput fahrenheit temperature: ");
+    scanfresult = scanf("%lf", &temp);
+    return scanfresult;
+}
+
+int main ()
```
Comment (dimmykar) [inline Chapter_5/9.c:29]: ditto

**Review by dimmykar — CHANGES_REQUESTED** [review]
Good, but you need to learn how to give clearer names for defines and variables and don't forget about `void`)

File: Chapter_5/1.c [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+int cycle (void)
+{
+    while (mins > 0)
+    {
+        float time1 = mins / X;
+        printf("\nTime in hours is %.3f", time1);
+        int a = (int) time1;
+        float b = time1 - a;
+        float c = (int) (b * 60);
+        printf("\nTime in hourse and minutes: %d hours %.0f minutes", a, c);
```
[the remark this answers is not in the document]
Comment (alexVOLTS) [reply Chapter_5/1.c:25]: LOL my mistake

File: Chapter_5/2.c [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+{
+    int number;
+    printf("Input INT number: ");
+    scanf("%d", &number);
+    
+    for (int i = 0; i < 10; i++)
+    {
+        printf("\t%d", number);
+        number++;
+    }
```
[the remark this answers is not in the document]
Comment (alexVOLTS) [reply Chapter_5/2.c:15]: Nice! Will note this one

File: Chapter_5/4.c [hunk 9]
```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <locale.h>
+#include <float.h>
+
+#define foot 0.0328084;
+#define inch 0.393701;
+
+float cm;
+
+int input()
```
[the remark this answers is not in the document]
Comment (alexVOLTS) [reply Chapter_5/4.c:10]: OK :D

**Pull request merged**